tests/testsuite/rustdoc.rs

```diff
@@ -20,6 +20,58 @@ fn rustdoc_simple() {
         .run();
 }
 
+#[cargo_test(nightly, reason = "--output-format is unstable")]
+fn rustdoc_simple_json() {
+    let p = project().file("src/lib.rs", "").build();
+
+    p.cargo("rustdoc -Z unstable-options --output-format json -v")
+        .masquerade_as_nightly_cargo(&["-Zunstable-options"])
+        .with_stderr(
+            "\
+[DOCUMENTING] foo v0.0.1 ([CWD])
+[RUNNING] `rustdoc [..]--crate-name foo src/lib.rs [..]\
+        -o [CWD]/target/doc \
+        [..] \
+        -L dependency=[CWD]/target/debug/deps [..]`
```

Comment:
Should assert `--output-format=json` here.

```suggestion
[RUNNING] `rustdoc [..]--crate-name foo [..]-o [CWD]/target/doc [..]--output-format=json[..]
```